src/tile-link/_tile-link-with-background.scss

```diff
@@ -53,5 +54,25 @@
       opacity: 1;
     }
   }
+
+  .tile-legal {
+    bottom: 1vw;
+    left: 1.5vw;
+    position: absolute;
+    color: $dark-grey;
+    font-size: 13px;
+    @include font($skytext-reg);
```

Comment:
I believe I was just trying to keep the font styles together, happy to move down tho.